src/Symfony/Bridge/Twig/Mime/Renderer.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Mime;
+
+use League\HTMLToMarkdown\HtmlConverter;
+use Twig\Environment;
+
+class Renderer
```

Comment:
Do you want to add `@author` tags to the classes?

Reply:
I don't think this is needed, author information is already present in the Git commit. In fact, many projects removed these.

Reply:
Sure, but in this PR some classes have and others does not have some. It should be consistent

Reply:
the class having it (with other names the Fabien generally) are classes for which the implementation comes from Swiftmailer (which is not reflected in the git history)